plugins/kube/apiclient.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"bytes"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/crunchydata/crunchy-watch/util"
+)
+
+
+func deletePrimaryPod(namespace string, name string ) error {
+
+	podsClient := client.CoreV1().Pods(namespace)
+	err := podsClient.Delete(name, nil)
+
+	if err != nil {
+
+		log.Error(err.Error())
+	}
+
+	return err
+}
+
+/*
+get replica by namespace and name
+ */
+func promoteReplica(namespace string, name string) error {
+
+
+	pod, err := client.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
+
+	if err != nil {
+		return  fmt.Errorf("could not get pod info: %v", err)
+	}
+
+	if len(pod.Spec.Containers) != 1 {
+		return  fmt.Errorf("could not determine which container to use")
+	}
+
+	log.Info("using exec")
```

Comment:
Can we be more specific with this log message? Is it really an Info message or should it be a Debug?